cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+function cleanup() {
+  if [ $? -ne 0 ]; then
+    rm -f "$CUSTOMIZE_IMAGE_PATH"
+  fi
+
+  rm -rf "$CLOUD_INIT_ISO"
+  virsh destroy "$DOMAIN_NAME" || true
```

Comment:
all over, please keep one format ie
"${VAR}" and not just double quote, since you did use with `"{` in some places and indeed its the cleanest

Reply:
Done
All scripts aligned to the format `"${VAR}"`